src/context/WalletProvider/MetaMask/components/MetaMaskMenu.tsx

```diff
@@ -44,6 +49,12 @@ export const MetaMaskMenu = () => {
 
   return isMetaMask ? (
     <>
+      {isSnapInstalled && (
```

Comment:
@NeOMakinG this already guards re: snap installed, so assuming we can't get into the account management flow without MM, we probably wouldn't need snap checks in `<ImportAccounts />`.
Now, there's a risk of uninstalling the snap during the import accounts flow, which is pretty edge-case but valid but meh

Reply:
I can confirm, uninstalling the snap while having the managing account window now disable the load more button due to the changes requested